src/bandmap.c

```diff
@@ -656,6 +656,15 @@ freq_t bm_get_center(int band, int mode) {
     return centerfrequency;
 }
 
+/* small helpers for filter_spots() */
+bool band_matches(spot *data) {
```

Comment:
unless used elsewhere these could be static to allow inlining by compiler

Reply:
Yes, good idea.